src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -292,22 +298,22 @@ void ThrusterDynamicEffector::computeForceTorque(double integTime, double timeSt
         }
 
         //! - For each thruster, aggregate the current thrust direction into composite body force
-        tmpThrustMag = it->MaxThrust*ops->ThrustFactor;
+        tmpThrustMag = it->MaxThrust * ops->ThrustFactor * ops->BlowDownFactor;
         // Apply dispersion to magnitude
         tmpThrustMag *= (1. + it->thrusterMagDisp);
         SingleThrusterForce = tmpThrustMag * thrustDirection_B;
         this->forceExternal_B += SingleThrusterForce;
 
         //! - Compute the point B relative torque and aggregate into the composite body torque
-        SingleThrusterTorque = thrustLocation_B.cross(SingleThrusterForce) + ops->ThrustFactor * it->MaxSwirlTorque * thrustDirection_B;
+        SingleThrusterTorque = thrustLocation_B.cross(SingleThrusterForce) + ops->ThrustFactor * ops->BlowDownFactor * it->MaxSwirlTorque * thrustDirection_B;
         this->torqueExternalPntB_B += SingleThrusterTorque;
 
 		if (!it->updateOnly) {
 			//! - Add the mass depletion force contribution
 			mDotNozzle = 0.0;
 			if (it->steadyIsp * ops->IspFactor > 0.0)
 			{
-				mDotNozzle = it->MaxThrust*ops->ThrustFactor / (EARTH_GRAV *
+				mDotNozzle = it->MaxThrust * ops->ThrustFactor * ops->BlowDownFactor / (EARTH_GRAV *
```

Comment:
can it->MaxThrust * ops->ThrustFactor * ops->BlowDownFactor be replaced by SingleThrusterForce?

Reply:
Good point, it can't be replaced by SingleThrusterForce but it can be replaced by tmpThrustMag for cleaner code so I've done that.